README.md

```diff
@@ -540,6 +540,18 @@ db.update({ _id: 'id6' }, { $push: { fruits: { $each: ['banana', 'orange'] } } }
 db.update({ _id: 'id6' }, { $push: { fruits: { $each: ['banana'], $slice: 2 } } }, {}, function () {
   // Now the fruits array is ['apple', 'orange']
 });
+
+// Update the value of the field, only if specified field is smaller or greater than the current value of the field
+// Let's say the database contains this document
+// doc = { _id: 'id1', name: 'Name', value: 5 }
+// $min can be used to update field if provided value is less than the current document value 
```

Comment:
There is an extra space at the end of the line, which shows up in red in my editor :)